scripts/globals/items/scroll_of_instant_protect.lua

```diff
@@ -0,0 +1,32 @@
+-----------------------------------
+-- ID: 5988
+--  Scroll of Instant Protect
+--  Grants the effect of Protect
+-----------------------------------
+require("scripts/globals/status")
+require("scripts/globals/msg")
+-----------------------------------
+local item_object = {}
+
+item_object.onItemCheck = function(target)
+    return 0
+end
+
+item_object.onItemUse = function(target)
+    local power = 1
+    local duration = 1800
+    if (target:hasStatusEffect(xi.effect.PROTECT)) then
+        local effect = target:getStatusEffect(xi.effect.PROTECT)
+        local oPower = effect:getPower()
+        if (oPower > power) then
```

Comment:
https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md#no-parentheses-unless-needed-to-clarify-order-of-operations